stdlib/universe/holt_winters_test.go

```diff
@@ -856,16 +858,16 @@ func TestHoltWinters_Process(t *testing.T) {
 					{Label: "_value", Type: flux.TFloat},
 				},
 				Data: [][]interface{}{
-					{execute.Time(1440713580000000000), 6.514215038632055},
-					{execute.Time(1440736320000000000), 8.497982411207028},
-					{execute.Time(1440759060000000000), 11.634451346818643},
-					{execute.Time(1440781800000000000), 16.593649776280504},
-					{execute.Time(1440804540000000000), 24.43508329162218},
-					{execute.Time(1440827280000000000), 36.834133612921335},
-					{execute.Time(1440850020000000000), 56.440059666607354},
-					{execute.Time(1440872760000000000), 87.44210023894752},
-					{execute.Time(1440895500000000000), 136.46464506566383},
-					{execute.Time(1440918240000000000), 213.98275775223067},
+					{execute.Time(1440713580000000000), 6.517746679116747},
+					{execute.Time(1440736320000000000), 8.44990936365862},
+					{execute.Time(1440759060000000000), 11.450103919073781},
+					{execute.Time(1440781800000000000), 16.108703181906527},
+					{execute.Time(1440804540000000000), 23.342418185768715},
+					{execute.Time(1440827280000000000), 34.57468700800914},
+					{execute.Time(1440850020000000000), 52.01577652114399},
+					{execute.Time(1440872760000000000), 79.09771500282905},
+					{execute.Time(1440895500000000000), 121.14964091593322},
+					{execute.Time(1440918240000000000), 186.4464618626079},
```

Comment:
Is it possible to add the SSE to these test cases so we can see that it improved as part of the diff?